simplq/src/components/pages/Status/QueueDetails.jsx

```diff
@@ -40,18 +23,7 @@ export default (props) => {
       expandable
       loading={!queueStatusResponse}
     >
-      <div className={styles['detail']}>
-        <DetailRow
-          title="People currently in queue:"
-          value={queueStatusResponse?.numberOfActiveTokens}
-          large
-        />
-        <DetailRow title="Queue creation time:" value={creationTime} />
-        <DetailRow
-          title="Total number of people joined in queue:"
-          value={queueStatusResponse?.totalNumberOfTokens}
-        />
-      </div>
+      <QueueStats queueStatus={queueStatusResponse} />
```

Comment:
Since you moved this component out, please also remove the styles specific to QueueDetails in status.module.scss